src/include/daos_fs.h

```diff
@@ -157,6 +157,56 @@ dfs_connect(const char *pool, const char *sys, const char *cont, int flags, dfs_
 int
 dfs_disconnect(dfs_t *dfs);
 
+/**
+ * Evict all the container handles from the hashtable and destroy the container.
+ *
+ * \param[in]   pool    Pool label where the container is.
+ * \param[in]	sys	DAOS system name to use for the pool.
+ *			Pass NULL to use the default system.
+ * \param[in]   cont    Container label to destroy.
+ * \param[in]	force	Container destroy will return failure if the container
+ *			is still busy (outstanding open handles from other open calls).
+ *			This parameter will force the destroy to proceed even if there is an
+ *			outstanding open handle.
+ * \param[in]	ev	Completion event, it is optional and can be NULL.
+ *			The function will run in blocking mode if \a ev is NULL.
+ *
+ * \return		0 on success, errno code on failure.
+ */
+int
+dfs_destroy(const char *pool, const char *sys, const char *cont, int force, daos_event_t *ev);
```

Comment:
we cannot create or destroy pools from the user API.
not sure what you mean by the second question. can you clarify?

Reply:
My first thought about the name `dfs_destroy` is that it would destroy the mountpoint or pool (not possible). I.e. something broader than just a container. From a different angle, comparing these function names side-by-side
- `dfs_cont_destroy(daos_handle_t poh, const char *cont, int force, daos_event_t *ev);`
  - destroys the container
- `dfs_destroy(const char *pool, const char *sys, const char *cont, int force, daos_event_t *ev);`
  - destroys the container AND evicts handles

I could see some user confusion about which to use.

Reply:
my thought is that dfs_connect/disconnect/destroy is 1 way to do things, and
the other APIs (dfs_pool/cont_connect/open/disconnect/close/destro; dfs_mount/umount) is the other.

if you have other suggestions on how to avoid confusion, please let me know

Reply:
The only thing that comes to mind with the current naming convention would be to split into:
```
dfs_evict("my_cont")
dfs_cont_destroy("my_cont")
```
But that's not necessarily less confusing.